src/store/selectors/songsQueue.js

```diff
@@ -18,3 +18,9 @@ export const selectSongPathAtIndex = (state, index) => {
   const songsQueue = selectSongsQueue(state);
   return songsQueue.getIn([index, 'path_lower']);
 };
+
+export const getPlayingSongPath = (state) => {
+  const index = selectIndex(state);
+  const songsQueue = selectSongsQueue(state);
+  return songsQueue.getIn([index, 'path_lower']);
+};
```

Comment:
DRY... literally you are duplicating the code from the function two lines above:

```js
export const getPlayingSongPath = (state) => {
  const index = selectIndex(state);
  return selectSongPathAtIndex(state, index);
};
```